src/routes/index.js

```diff
@@ -18,12 +18,34 @@ export default function BaseRouter() {
     };
   });
 
+  const checkRoute = (route, index) => {
+    if (isLoggedIn && route.permissions === isProtectedRoute) {
+      return <Route key={index} component={() => route.component()} exact path={route.path} />;
+    }
+    if (!isLoggedIn && route.permissions === isPublicRoute) {
+      return <Route key={index} component={() => route.component()} exact path={route.path} />;
+    }
+
+    if (isLoggedIn) return <Route key={index} component={() => <HomeContainer />} exact path="/home" />;
+    return <Route key={index} component={() => <LoginContainer />} exact path="/login" />;
+  };
+
   return (
-    <Switch>
-      {RouteConfig.auth.map((route, index) => {
-        return <Route key={index} component={() => <AuthRoute route={route} />} exact path={route.path} />;
-      })}
-      {isLoggedIn ? <Route component={Home} /> : <Route component={LoginContainer} />}
-    </Switch>
+    <>
+      <Switch>
+        {RouteConfig.auth.map((route, index) => {
+          return checkRoute(route, index);
+        })}
+        {isLoggedIn ? (
```

Comment:
Why we are doing this ??

Reply:
For dashboard routes and other private routes we must use the RouteConfigs array as we are using for auth